source/raylib/rcore.d

```diff
@@ -11,6 +11,7 @@ import raylib.external.sdefl;
 
 import core.stdc.stdlib;
 import core.stdc.math;
+import core.sys.posix.time;
```

Comment:
If this is needed to build on Linux, you need to put this in the `version(Posix)` block below.

Reply:
I'll be honest, I don't even know how that got here. It may be leftovers of an older version of draylib, which is strange because I thought I had merged down before making that change.